python/lsst/analysis/drp/calcFunctors.py

```diff
@@ -71,6 +75,39 @@ def __call__(self, df):
         return magDiff
 
 
+class ExtinctionCorrectedMagDiff(DataFrameAction):
+    magDiff = ConfigurableActionField(doc="Action that returns a difference in magnitudes",
+                                          default=MagDiff, dtype=DataFrameAction)
+    ebv = Field(doc="E(B-V) Column", dtype=str, default="ebv")
```

Comment:
I think this should be `colEbv` or something like that, because it's a column not a value.